addon/helpers/import-export.js

```diff
@@ -28,16 +29,19 @@ export function importData(store, content, options) {
 
       adapter._getIndex(type).reset();
 
-      // unload from store
-      store.unloadAll(singularize(type));
+      truncateTypes.addObject(singularize(type));
```

Comment:
Can we change this to:

```js
    content.data.forEach((record) => {
      truncateTypes.addObject(record.type);
    });
    
    truncateTypes.forEach((type) => {
      const singularType = singularize(type);
      const adapter = store.adapterFor(singularType);

      adapter._getIndex(type).forEach((storageKey) => {
        delete get(adapter, '_storage')[storageKey];
      });

      adapter._getIndex(type).reset();

      // unload from store
      store.unloadAll(singularType);
    });
```

Reply:
So we don't do the work of reseting the index more than needed.

Reply:
Good catch, completely overlooked that, cheers!